scripts/word-list.php

```diff
@@ -0,0 +1,61 @@
+<?php
```

Comment:
Hi, thanks for the PR. What is the purpose of this file?

Reply:
Sry, it is temp file... We use php to convert one czech dictionary to plain text ;(